### PR TITLE
Maintenance/277/update dependencies

### LINKED ISSUE #277: Update dependencies
Heya, I'd like to keep dependencies up to date in some downstream projects, so would like to bump some dependencies in this crate.

I've got a PR incoming.

### PR DESCRIPTION
**Pull request by azriel91**
Closes #277.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ The `gltf` crate adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Changed
```
Comment (alteous) [inline CHANGELOG.md]: Thank you :)

**Comment by sunjay** [comment]
@alteous Could a new release be made with the updated dependencies? I ran into an issue today because I am using the latest release of `image`, but the published `gltf` version does not. (This PR did update everything to be up-to-date.)

**Comment by alteous** [comment]
Sure, I'll get around to it this weekend.

**Comment by aloucks** [comment]
> Sure, I'll get around to it this weekend.

@alteous Could you merge #280 first? Thanks!

**Comment by alteous** [comment]
I have substantially more time to work on side projects in the next two weeks so there's no need to rush #280 in.

**Pull request merged**